### PR TITLE
TBRANDS-36: Default Output Block

### PR DESCRIPTION
**Pull request by nschubach**
## Description

Update default-output-block to new advanced styles.

## Jira Ticket

- [TBRANDS-36](https://arcpublishing.atlassian.net/browse/TBRANDS-36)

## Acceptance Criteria

The existing Default Output Type block can be used for the Commerce Theme 

## Test Steps

1. Checkout this branch `git checkout tbrands-36`
2. Modify `arc-themes-feature-pack/.env` and set `FUSION_RELEASE=3.3.0-enhanced-styling.2`
3. Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add `@use "@wpmedia/default-output-block";` (https://github.com/WPMedia/arc-themes-feature-pack/pull/213)
4. Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add
```
	$tokens: (
		default: (
			blocks: (
				application: (
					background-color: var(--global-neutral-1),
				),
			),
```
5. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/default-output-block`
6. Open `localhost/pagebuilder/pages`
7. Make sure the Output Type at the top of the page is `default`
image
8. Create a page (name and URI not important)
9. Use the Single Column Regular Width Layout that's available (not really important which layout)
10. Share and publish
11. Preview and Inspect the page
12. See *After* below for screen captures of changes
13. View `localhost/pf/dist/css/the-gazette.css?d=%24LATEST`
14. Validate the following CSS exists:
```
.b-application {
  background-color: var(--b-application-background-color);
}
```

### Before

N/A

### After

image

## Dependencies or Side Effects

Modify `arc-themes-feature-pack/.env` and set `FUSION_RELEASE=3.3.0-enhanced-styling.2`
Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add `@use "@wpmedia/default-output-block";` (https://github.com/WPMedia/arc-themes-feature-pack/pull/213)
Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add
```
	$tokens: (
		default: (
			blocks: (
				application: (
					background-color: var(--global-neutral-1),
				),
			),
```

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/default-output-block/output-types/default.jsx [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
+import React, { Fragment } from "react";
 import getProperties from "fusion:properties";
 import getTranslatedPhrases from "fusion:intl";
 import { useFusionContext } from "fusion:context";
 import { MetaData } from "@wpmedia/engine-theme-sdk";
+import { Stack } from "@wpmedia/arc-themes-components";
 
 import blocks from "~/blocks.json";
-// this is blank import but used to inject scss
-import "./default.scss";
```
Comment (JackHowa) [inline blocks/default-output-block/output-types/default.jsx:9]: was this deleted deliberately? I know that this is used for injecting scss

File: blocks/default-output-block/index.js [hunk 2]
```diff
@@ -1 +0,0 @@
-module.exports = {};
```
Comment (JackHowa) [inline blocks/default-output-block/index.js:1]: is this change in the scope of this ticket? I know there were some potential side effects of doing this for feeds

**Review by JackHowa — COMMENTED** [review]
Seems to be doing what's described. But it looks like a lot of other work was done. The key changes aren't too destructive. But I'd be concerned about non-deliberate change for the index.js and the importing of the blank scss 

Screen Shot 2022-04-05 at 09 38 19

File: blocks/default-output-block/output-types/default.jsx (hunk 1 above)
Comment (JackHowa) [reply blocks/default-output-block/output-types/default.jsx:9]: this is ok to remove

Comment (nschubach) [reply blocks/default-output-block/output-types/default.jsx:9]: Not necessary in v2

File: blocks/default-output-block/index.js (hunk 2 above)
Comment (JackHowa) [reply blocks/default-output-block/index.js:1]: this is ok to remove then based on conversation

**Review by JackHowa — APPROVED** [review]
Based on conversation, the other changes are good to remove

File: blocks/default-output-block/index.js (hunk 2 above)
Comment (nschubach) [reply blocks/default-output-block/index.js:1]: Yep.  The new themes needs to import the _index.scss and with the index.js there it doesn't find it.  We'll be removing this for most blocks.

**Pull request merged**